pkg/sentry/socket/unix/transport/unix.go

```diff
@@ -187,7 +196,7 @@ type Endpoint interface {
 	//
 	// peerAddr if not nil will be populated with the address of the connected
 	// peer on a successful accept.
-	Accept(ctx context.Context, peerAddr *Address) (Endpoint, *syserr.Error)
+	Accept(ctx context.Context, peerAddr *Address, opts UnixSocketOpts) (Endpoint, *syserr.Error)
```

Comment:
Same as above: it looks like this is the only method that actually uses `UnixSocketOpts`, so maybe we should remove it from the other methods.